lib/features/diary_detail/bloc/diary_detail_cubit.dart

```diff
@@ -0,0 +1,138 @@
+        'attributes': {'color': '#0fabe2'}
+      },
+      {'insert': '\n\n'},
+      {
+        'insert':
+            'Steak anchovies parmesan ipsum white ipsum personal string platter. White platter ipsum ',
+        'attributes': {'italic': true}
+      },
+      {
+        'insert':
+            'lasagna wing style ricotta. Bell white thin platter thin bacon Chicago. '
+      },
+      {
+        'insert': 'Mayo sausage NY green lasagna Aussie deep roll bacon. ',
+        'attributes': {'bold': true, 'italic': true, 'underline': true}
+      },
+      {
+        'insert':
+            'Ricotta thin and large pork lovers. Dolor pizza personal green broccoli green Aussie pesto melted black. Banana broccoli spinach meat meat sautéed bell. '
+      },
+      {
+        'insert':
+            'Pesto bacon pepperoni sausage wing mozzarella. Pie mayo meat pesto and pepperoni dolor dolor thin.',
+        'attributes': {'background': '#2f89eb', 'color': '#faa49e'}
+      },
+      {'insert': '\n\n'},
+      {
+        'insert': 'Lorem ipsum dolor sit amet',
+        'attributes': {'color': '#0fabe2'}
+      },
+      {'insert': '\n'},
+      {
+        'insert':
+            'Consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore.',
+        'attributes': {'color': '#00eeff'}
+      },
+      {'insert': '\n\n'},
+      {
+        'insert':
+            'lasagna wing style ricotta. Bell white thin platter thin bacon Chicago. '
+      },
+      {
+        'insert': 'Mayo sausage NY green lasagna Aussie deep roll bacon. ',
+        'attributes': {'bold': true, 'italic': true, 'underline': true}
+      },
+      {
+        'insert':
+            'Ricotta thin and large pork lovers. Dolor pizza personal green broccoli green Aussie pesto melted black. Banana broccoli spinach meat meat sautéed bell. '
+      },
+      {
+        'insert':
+            'Pesto bacon pepperoni sausage wing mozzarella. Pie mayo meat pesto and pepperoni dolor dolor thin.',
+        'attributes': {'background': '#2f89eb', 'color': '#faa49e'}
+      },
+      {
+        'insert':
+            'Steak anchovies parmesan ipsum white ipsum personal string platter. White platter ipsum ',
+        'attributes': {'italic': true}
+      },
+      {
+        'insert':
+            'lasagna wing style ricotta. Bell white thin platter thin bacon Chicago. '
+      },
+      {
+        'insert': 'Mayo sausage NY green lasagna Aussie deep roll bacon. ',
+        'attributes': {'bold': true, 'italic': true, 'underline': true}
+      },
+      {
+        'insert':
+            'Ricotta thin and large pork lovers. Dolor pizza personal green broccoli green Aussie pesto melted black. Banana broccoli spinach meat meat sautéed bell. '
+      },
+      {'insert': '\n\n'},
+    ];
+    Document? document = Document.fromJson(textJson);
+    String plainText = document.toPlainText();
+    QuillController richText = QuillController(
```

Comment:
QuillController is a UI component, it should not be in a cubit, you can pass the json text from db to UI, and use Document and QuillController on UI to display